google-cloud-language/lib/google/cloud/language.rb

```diff
@@ -54,10 +54,10 @@ module Cloud
     # ## Creating documents
     #
     # Cloud Natural Language API supports UTF-8, UTF-16, and UTF-32 encodings.
```

Comment:
This line is ambiguous whether it means requests support these encodings or responses. To be clear, request can be anything since the proto converter will convert it to utf8 before sending. The responses can be any utf8 utf16 or utf32 based on default internal encoding.